addons/l10n_latam_account_sequence/models/account_move.py

```diff
@@ -20,7 +20,7 @@ def _auto_init(self):
 
             # Fixup the account.move names like "sequence (N)" removing the "(N)" part
             self.env.cr.execute("""
-            UPDATE account_move SET name = SUBSTRING(account_move.name, 1, strpos(account_move.name::varchar, ' ('::varchar) -1 )
+            UPDATE account_move SET name = RTRIM(SUBSTRING(account_move.name, 1, strpos(account_move.name::varchar, '('::varchar) -1 ))
              WHERE l10n_latam_document_type_id IS NOT NULL AND account_move.name LIKE '%(%)'
```

Comment:
```suggestion
             WHERE l10n_latam_document_type_id IS NOT NULL AND account_move.name LIKE '% (%)'
```
This thing is to counteract what is being done in account_sequence and if there are doubles it adds it with the space, so we should only remove it when it is that script that added it.  So, I would suggest this change instead.  (although this case should never happen anyways in theory)

Reply:
Updated.
(In practice, this case is blocking some upgrades to Odoo 16)